frontend/src/Student/components/ScholarshipAdminForm.js

```diff
@@ -0,0 +1,151 @@
+import React, { Fragment } from 'react'
```

Comment:
@moiseslorap Didn't we say we'd move this to the admin folder?
https://github.com/johnbyrne727rit/gallery-gateway/blob/c75ecfe0fff5e53d919e6eef50bc899bb8949074/frontend/src/Admin/components/ScholarshipForm.js#L1